plugins/source/aws/resources/services/ec2/instance_credit_specifications.go

```diff
@@ -0,0 +1,42 @@
+package ec2
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func InstanceCreditSpecifications() *schema.Table {
+	tableName := "aws_ec2_instance_credit_specifications"
+	return &schema.Table{
+		Name:        tableName,
+		Description: `https://docs.aws.amazon.com/sdk-for-go/api/service/ec2/#EC2.DescribeInstanceCreditSpecifications`,
```

Comment:
Actually, comparing against the other descriptions it should probably be this:

```suggestion
		Description: `https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_InstanceCreditSpecification.html`,
```